projects/pgai/pgai/vectorizer/loading.py

```diff
@@ -43,18 +44,51 @@ class UriLoading(BaseModel):
     implementation: Literal["uri"]
     column_name: str
     retries: int = 6
+    aws_role_arn: str | None = None
 
     def load(self, row: dict[str, str]) -> LoadedDocument:
         # Note: deferred import to avoid import overhead
+        import boto3
```

Comment:
since boto is listed as a project dependency anyway, do we want to simply put it at the top of the file?

Reply:
see https://github.com/timescale/pgai/pull/606 for a rationale on why this is here.